planarenvs/ground_robots/envs/acc.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
+from gym import spaces
+
+from planarenvs.ground_robots.envs.ground_robot_env import GroundRobotEnv
+
+
+class GroundRobotAccEnv(GroundRobotEnv):
+    def set_spaces(self):
+        self._observation_space = spaces.Dict(
+            {
+                "x": spaces.Box(
+                    low=-self._limUpPos, high=self._limUpPos, dtype=np.float64
+                ),
+                "xdot": spaces.Box(
+                    low=-self._limUpVel, high=self._limUpVel, dtype=np.float64
+                ),
+                "vel": spaces.Box(
```

Comment:
What is the difference between xdot and vel? I feel like a little more explanation would help here, since it is quite relevant for using the environment properly.

Reply:
For differential drives, you have `xdot` as velocity in x, y and theta direction and you have the actuated velocity, meaning forward and angular velocity of the base. Note that theta_dot is the same as angular velocity.
I'll add some documentation to it.